.github/ISSUE_TEMPLATE/plugin_request.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Plugin Request
+description: Request support for a new plugin
+title: "[PLUGIN REQUEST] <PLUGIN>"
+labels: ["enhancement"]
```

Comment:
Adding a bespoke "plugin request" label here would help us simplify our Plugin Requests project board...

Currently it adds issues with matching titles, but that's kinda hanky.

This matches the existing behaviour though, so I'm just noting this for a future PR.